src/app/qgsrasterlayerproperties.cpp

```diff
@@ -1066,7 +1066,12 @@ void QgsRasterLayerProperties::apply()
 
 void QgsRasterLayerProperties::mLayerOrigNameLineEd_textEdited( const QString &text )
 {
-  leDisplayName->setText( mRasterLayer->formatLayerName( text ) );
+  QgsSettings settings;
```

Comment:
I think this one is safe to leave. What do you think?

Reply:
@nyalldawson , what do you mean? Remove the formatting altogether?